app/models/concerns/post_concerns/autoflagging.rb

```diff
@@ -52,10 +52,13 @@ def autoflag
         Rails.logger.warn "[autoflagging] #{id}: scaled maxes: #{scaled_maxes}"
         if !scaled_maxes.nil? && scaled_maxes.count == 6
           # Check historical accuracy
+          post_reasons_count = 0
+          post.reasons.each do |r|
+            post_reasons_count += 1
           fake_flag_condition = FlagCondition.new(
             sites: Site.mains,
             max_poster_rep: post.user_reputation,
-            min_reason_count: 1,
+            min_reason_count: post_reasons_count,
```

Comment:
Ruby is not Python. You're writing Python-style Ruby, and... it's not going to work. You want `post.reasons.count`

Reply:
Well, I used `post.reasons.count`, and CI is not happy about it...